SPECS/python-sphinxcontrib-serializinghtml/python-sphinxcontrib-serializinghtml.spec

```diff
@@ -2,22 +2,24 @@
 
 Summary:        Sphinx extension for serialized HTML
 Name:           python-%{pypi_name}
-Version:        1.1.5
-Release:        5%{?dist}
+Version:        1.1.10
+Release:        1%{?dist}
 License:        BSD
 Vendor:         Microsoft Corporation
 Distribution:   Azure Linux
 URL:            https://www.sphinx-doc.org/en/master/
-Source0:        https://files.pythonhosted.org/packages/b5/72/835d6fadb9e5d02304cf39b18f93d227cd93abd3c41ebf58e6853eeb1455/%{pypi_name}-%{version}.tar.gz
+Source0:        https://files.pythonhosted.org/packages/54/13/8dd7a7ed9c58e16e20c7f4ce8e4cb6943eb580955236d0c0d00079a73c49/sphinxcontrib_serializinghtml-1.1.10.tar.gz#/%{pypi_name}-%{version}.tar.gz
```

Comment:
nit: why the change away from ${pypi_name} and %{version}?

Reply:
because of how the tar file is named when unzipped
it's a dash vs underscore thing
%version here won't add anything as the link would change entirely when upgrading since it has a unique hash in it

Reply:
it seems like the version could still be used?

Reply:
updated to use it

Reply:
i think i would suggest introducing a new variable like %global pypi_name sphinxcontrib-serializinghtml, except with the underscore ... and use that in the Source and the autosetup line.  could be something like:
 
%global pypi_name_prefix sphinxcontrib
%global pypi_name_suffix serializinghtml
%global pypi_name %{pypi_name_prefix}-${pypi_name_suffix}
%global pypi_name_underscore %{pypi_name_prefix}_${pypi_name_suffix}